src/slider/_slider.scss

```diff
@@ -190,24 +190,17 @@ _:-ms-input-placeholder, :root .mdl-slider.mdl-slider.is-upgraded {
     }
 
     &.is-lowest-value:focus:not(:active)::-webkit-slider-thumb {
-      border: 1.8px solid $range-bg-color;
-      transform: scale(1.33);
-      box-shadow: none;
+      box-shadow: 0 0 0 10px $range-bg-focus-color;
+      background: $range-bg-focus-color;
     }
 
     &.is-lowest-value:focus:not(:active)::-moz-range-thumb {
-      border: 1.8px solid $range-bg-color;
-      transform: scale(1.33);
-      box-shadow: none;
-    }
-
-    &.is-lowest-value:focus:not(:active) ~
-        .mdl-slider__background-flex > .mdl-slider__background-upper {
-      left: 8px;
+      box-shadow: 0 0 0 10px $range-bg-focus-color;
+      background: $range-bg-focus-color;
```

Comment:
Nah, if the rest of the file is written the same way, it’s fine. Maybe at some point we can combine all the prefixed values.